copy file instead of renaming it.

RMastop opened the pull request:
As stated in thread; fixes #5050

Please issue pull request against the `3.x-dev` branch only.

Piwik 2 is in LTS mode. This means we do not accept any pull request for 2.x except critical security bugs and major data loss bugs. 

If you need to create a pull request for 2.x, then please also create the pull request against the `3.x-dev` so we can merge both.

Happy hacking!

RMastop committed:
copy file instead of renaming it.

As stated in thread; fixes #5050

mattab commented on plugins/UserCountry/GeoIPAutoUpdater.php:
```diff
@@ -268,7 +268,7 @@ public static function unzipDownloadedFile($path, $unlink = false)
 
             $tempFile = GeoIp::getPathForGeoIpDatabase($tempFilename);
-            rename($tempFile, $oldDbFile);
+            copy($tempFile, $oldDbFile);
```
Hi Richard, should we also delete the `$tempFile` then?
have you had a chance to test this works?

RMastop committed:
Update GeoIPAutoUpdater.php

In most cases the rename is successful, in rare cases the rename cannot be completed. In those cases we do a copy, leaving the $tempFile in place, it will be overwritten during the next  GeoIPAutopudater task.

Tested in both windows and Ubuntu environment.

mattab committed:
Add unlink
